doc/whats_new/v0.24.rst

```diff
@@ -364,7 +367,7 @@ Changelog
 - |Enhancement| :class:`multiclass.OneVsOneClassifier` now accepts
   the inputs with missing values. Hence, estimators which can handle
   missing values (may be a pipeline with imputation step) can be used as
-  a estimator for multiclass wrappers. 
+  a estimator for multiclass wrappers.
```

Comment:
Good catch (here and below)!

Reply:
This was done with a pre-commit hook. (I normally revert these types of commits, but these slipped through).